commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -306,10 +297,22 @@ def _prepare_segment_event_properties(in_order, return_line_item_return_id):
 
     logger.info(f'[CT-{tag}] calling PSP to refund payment "{psp_payment_id}", message id: {message_id}')
 
+    return_line_item_ids = [line_item_return['lineItemId'] for line_item_return in return_items]
+    return_line_item_return_dict = {
+        line_item_return['lineItemId']: line_item_return['id'] for line_item_return in return_items}
+    return_line_item_return_ids = [line_item_return['id'] for line_item_return in return_items]
+    return_line_entitlement_ids = {
+        return_line_item_return_dict.get(line_item.id): get_line_item_lms_entitlement_id(line_item)
+        for line_item in get_edx_items(order)}
```

Comment:
this has become tricky code, could you please add small comments for reader's easyness?

also some code restructuring suggestion

```suggestion
return_line_item_ids = []
return_line_item_return_dict = {}
return_line_item_return_ids = []

for item in return_items:
    line_item_id = item["lineItemId"]
    return_id = item["id"]
    
    return_line_item_ids.append(line_item_id)
    return_line_item_return_dict[line_item_id] = return_id
    return_line_item_return_ids.append(return_id)

return_line_entitlement_ids = {
    return_line_item_return_dict.get(line_item.id): get_line_item_lms_entitlement_id(line_item)
    for line_item in get_edx_items(order)
} 
```

Reply:
To me this looks more readable 👀 
```
return_line_item_return_dict = {
    item["lineItemId"]: item["id"] for item in return_items
}
return_line_entitlement_ids_dict = {
    return_line_item_return_dict.get(line_item.id): get_line_item_lms_entitlement_id(line_item)
    for line_item in get_edx_items(order)
}

return_line_item_ids = list(return_line_item_return_dict.keys())
return_line_item_return_ids = list(return_line_item_return_dict.values())
```